File: YesGraph/Example/ExampleUITests/YSGTestMockData.m
//
//  YSGTestMockData.m
//  YesGraphSDK
//
//  Created by Dal Rupnik on 27/08/15.
//  Copyright © 2015 YesGraph. All rights reserved.
//

#import "YSGTestMockData.h"

@implementation YSGTestMockData

+ (YSGContactList *)mockContactList
{
    YSGContactList *contactList = [[YSGContactList alloc] init];
    NSDictionary *sourceDictionary1 = @
    {
        @"name": @"test name",
        @"email": @"test.email@string.com",
        @"phone": @"+386 01 111 111",
        @"type": @"not ios"
    };

    contactList.source = [YSGSource ysg_objectWithDictionary:sourceDictionary1];
    YSGContact *c1 = [YSGContact ysg_objectWithDictionary:@{ @"name": @"Daniel Higgins Jr.", @"emails:" : @[ @"d-higgins@mac.com"] }];
    YSGContact *c2 = [YSGContact ysg_objectWithDictionary:@{ @"name": @"Anna Haro", @"emails:" : @[ @"anna-haro@mac.com"] }];
    YSGContact *c3 = [YSGContact ysg_objectWithDictionary:@{ @"name": @"Hank M. Zakroff", @"emails:" : @[ @"hank-zakroff@mac.com"] }];
    YSGContact *c4 = [YSGContact ysg_objectWithDictionary:@{ @"name": @"Kate Bell", @"emails:" : @[ @"kate-bell@mac.com"] }];
    YSGContact *c5 = [YSGContact ysg_objectWithDictionary:@{ @"name": @"John Appleseed", @"emails:" : @[ @"John-Appleseed@mac.com"] }];
    YSGContact *c6 = [YSGContact ysg_objectWithDictionary:@{ @"name": @"David Taylor", @"phones:" : @[ @"555-610-6679"] }];
    
    contactList.entries = @[ c1, c2, c3, c4, c5, c6 ];
    
    return contactList;
}

@end

File: YesGraph/Example-Swift/Example-SwiftUITests/YSGTestMockData.swift
//
//  YSGTestMockData.swift
//  Example-Swift
//
//  Created by Nejc Vivod on 11/10/15.
//  Copyright © 2015 Dal Rupnik. All rights reserved.
//

import Foundation
import YesGraphSDK

class YSGTestMockData: NSObject {
    class var mockContactList : [YSGContact] {
        var retList = [YSGContact]();
        let c1 = YSGContact();
        c1.name = "Daniel Higgins Jr.";
        c1.emails = ["d-higginsmac.com"];
        retList.append(c1);
        
        let c2 = YSGContact();
        c2.name = "Anna Haro";
        c2.emails = ["anna-haromac.com"];
        retList.append(c2);
        
        let c3 = YSGContact();
        c3.name = "Hank M. Zakroff";
        c3.emails = ["hank-zakroffmac.com"];
        retList.append(c3);
        
        let c4 = YSGContact();
        c4.name = "Kate Bell";
        c4.emails = ["kate-bellmac.com"];
        retList.append(c4);
        
        let c5 = YSGContact();
        c5.name = "John Appleseed";
        c5.emails = ["John-Appleseedmac.com"];
        retList.append(c5);
        
        let c6 = YSGContact();
        c6.name = "David Taylor";
        c6.phones = ["555-610-6679"];
        retList.append(c6);
        
        return retList;
    }
}